quesma/plugins/ingest_validator/ingest_validator.go

```diff
@@ -0,0 +1,64 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+package ingest_validator
+
+import (
+	"quesma/clickhouse"
+	"quesma/logger"
+	"quesma/plugins"
+	"quesma/quesma/config"
+	"quesma/quesma/types"
+	"quesma/schema"
+	"quesma/util"
+	"reflect"
+)
+
+type IngestValidator struct {
+	cfg            config.QuesmaConfiguration
+	schemaRegistry schema.Registry
+	table          string
+	tableMap       clickhouse.TableMap
+}
+
+func (iv *IngestValidator) Transform(document types.JSON) (types.JSON, error) {
+	clickhouseTable, ok := iv.tableMap.Load(iv.table)
+	if !ok {
+		logger.Error().Msgf("Table %s not found", iv.table)
+		return document, nil
```

Comment:
I assume it will return an error here.

Reply:
Yes, but the implementation is incomplete yet. I plan to implement that in next PR